static/js/stream_popover.js

```diff
@@ -722,7 +722,7 @@ function with_first_message_id(stream_id, topic_name, success_cb, error_cb) {
             const message_id = data.messages[0].id;
             success_cb(message_id);
         },
-        error_cb,
+        error: error_cb,
```

Comment:
Just to be clear -- this was a bug where we were accidentally passing a parameter that didn't do anything, right?

Reply:
Yes.